src/txmempool.h

```diff
@@ -6,7 +6,6 @@
 #ifndef BITCOIN_TXMEMPOOL_H
 #define BITCOIN_TXMEMPOOL_H
 
-#include <list>
 #include <memory>
 #include <set>
```

Comment:
Added some missing includes (map for std::map, utility for std::pair, vector for std::vector, string for std::string). I'm sure we have many more cases of these missing though.